index.html

```diff
@@ -81,6 +81,209 @@ <h1 class="heading-primary">
+                <li class="work-feature-item">CSS</li>
+                <li class="work-feature-item">JavaScript</li>
+                <li class="work-feature-item">HTML</li>
+              </ul>
+              <a href="#" class="btn">See project</a>
+            </div>
+          </div>
+          <!-- Work 1 -->
+          <div class="work">
+            <div class="work-image-holder"></div>
+            <div class="work-details">
+              <h3 class="heading-tertiary">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h3>
+              <ul class="work-features">
+                <li class="work-feature-item">Ruby on Rails</li>
+                <li class="work-feature-item">CSS</li>
+                <li class="work-feature-item">JavaScript</li>
+                <li class="work-feature-item">HTML</li>
+              </ul>
+              <a href="#" class="btn">See project</a>
+            </div>
+          </div>
+          <!-- Work 1 -->
+          <div class="work">
+            <div class="work-image-holder"></div>
+            <div class="work-details">
+              <h3 class="heading-tertiary">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h3>
+              <ul class="work-features">
+                <li class="work-feature-item">Ruby on Rails</li>
+                <li class="work-feature-item">CSS</li>
+                <li class="work-feature-item">JavaScript</li>
+                <li class="work-feature-item">HTML</li>
+              </ul>
+              <a href="#" class="btn">See project</a>
+            </div>
+          </div>
+          <!-- Work 1 -->
+          <div class="work">
+            <div class="work-image-holder"></div>
+            <div class="work-details">
+              <h3 class="heading-tertiary">
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h3>
+              <ul class="work-features">
+                <li class="work-feature-item">Ruby on Rails</li>
+                <li class="work-feature-item">CSS</li>
+                <li class="work-feature-item">JavaScript</li>
+                <li class="work-feature-item">HTML</li>
+              </ul>
+              <a href="#" class="btn">See project</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!----------------------->
+      <!-- ABOUT ME SECTION -->
+      <!----------------------->
+
+      <section class="section-about-me">
+        <div class="about-me-header">
+          <h2 class="heading-secondary">About me</h2>
+          <p class="about-me-description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <a href="#" class="btn">Get my resume</a>
```

Comment:
- [ ] Kindly follow the button styling: Active, Hover, and Pressed 
- Figma Design
<img width="939" alt="Screenshot 2023-04-27 at 23 28 33" src="https://user-images.githubusercontent.com/81444338/235071603-ba45eb8d-0a62-4cbb-b048-37be877830f8.png">